src/index.html

```diff
@@ -5,8 +5,318 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__navigation">
+        <ul class="menu">
+          <li class="menu__list">
+            <a class="is-active menu__link" href="#">Apple</a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Samsung</a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="menu__list">
+            <a
+              class="menu__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Tablets</a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Photo</a>
+          </li>
+
+          <li class="menu__list">
+            <a class="menu__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="goods">
+      <article class="product" data-qa="card">
+        <img
+          class="product__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
+
+        <h2 class="product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="product__code">Product code: 195434</p>
+
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product__review">Reviews: 5</p>
+        </div>
+
+        <div class="product__price">
+          <p class="product__price-text">Price:</p>
+
+          <p class="product__cost">$2,199</p>
+        </div>
+
+        <a
+          class="product__buy"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+      <article class="product">
```

Comment:
Add the blank line between the cards
```suggestion
      </article>
      
      <article class="product">
```